README.md

```diff
@@ -1,28 +1,24 @@
-# SFS CSI Driver for Kubernetes
-Huawei Cloud SFS Container Storage Interface (CSI) Plugin `sfs.csi.huaweicloud.com`
-
-### Prerequisite
- - The driver initialization depends on a [cloud config file](./deploy/cloud-config). Make sure it's in `/etc/sfs/cloud-config` on your node.
-
-### Install SFS CSI driver
-
-```
-kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/deploy/sfs-csi-plugin/kubernetes/rbac-csi-sfs-controller.yaml
-kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/deploy/sfs-csi-plugin/kubernetes/rbac-csi-sfs-node.yaml
-kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/deploy/sfs-csi-plugin/kubernetes/csi-sfs-controller.yaml
-kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/deploy/sfs-csi-plugin/kubernetes/csi-sfs-node.yaml
-kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/deploy/sfs-csi-plugin/kubernetes/csi-sfs-driver.yaml
-```
-
-### Examples
-
-```
-kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/examples/sfs-csi-plugin/kubernetes/sc.yaml
-kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/examples/sfs-csi-plugin/kubernetes/pvc.yaml
-kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/examples/sfs-csi-plugin/kubernetes/pod.yaml
-```
-
-### Links
- - [Kubernetes CSI Documentation](https://kubernetes-csi.github.io/docs/Home.html)
- - [CSI Drivers](https://github.com/kubernetes-csi/drivers)
- - [Container Storage Interface (CSI) Specification](https://github.com/container-storage-interface/spec)
+# Huawei Cloud CSI Plugin
+
+English | [简体中文](./README_CN.md)
+
+Huawei cloud CSI Plugin connects the cloud storage service of the Huawei cloud platform. Users can easily access the
+Huawei cloud storage service through this plug-in.
+
+As a high-quality storage service provider, Huawei cloud provides complete supporting storage resources for users to
+choose, and can flexibly meet users' needs.
+
+Huawei cloud CSI Plugin currently provides the following capabilities, other storage capabilities are under continuous
+development.
+
+This Repository hosts various plugins relevant to Huawei Cloud and Kubernetes Integration
+
+## [EVS CSI Plugin](./docs/evs.md)

```

Comment:
do not use hyperlinks in chapter titles.
it is recommended to add the following:
See [EVS CSI Plugin](./docs/evs.md) for details.

Reply:
done